src/Camp25US.elm

```diff
@@ -0,0 +1,341 @@
+module Camp25US exposing (..)
+
+import Camp24Uk.Archive
```

Comment:
Let's get rid of these 2 imports.

Reply:
Maybe easiest to comment out here and lower down, rather than making new directories. We'll use it after the conference.